### ISSUE TITLE
Check/Verify Install dependencies

### ISSUE THREAD

**Issue by pbatroff** [systopia/de.systopia.eventcheckin#4]
When installing the Extension from the Release Tag ([download link](https://github.com/systopia/de.systopia.eventcheckin/archive/refs/tags/1.0-alpha2.tar.gz)) the [PHP QR Code generator](https://github.com/chillerlan/php-qrcode) Library is missing.

When installing the extension without the Library a 500 Error is thrown. If possible a warning and install interrupt should be thrown instead if possible, similar to Other Extension Dependenacies.

**Comment by bjendres** [comment]
Yes, the automatically generated tarballs don't include the composer dependencies. The curated releases, however, do.

In any case it would be nice to add some code to warn the user, if the dependency isn't there.

**Comment by jensschuppe** [comment]
The readme contains a hint about composer dependencies.
